### ISSUE TITLE
Cannot edit text sections on certain pages

### ISSUE THREAD

**Issue by mikedecosta** [elementor/elementor#15672]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

* The edit page will load but prior to fully loading, the dev console will print a bunch of errors.
* Trying to edit a text box by clicking the edit pencil icon doesn't do anything and the console prints 2 more errors.
* Interacting with other elements still works correctly.
* This doesn't happen on all pages

### Error Screenshot
![Screen Shot 2021-07-15 at 8 20 27 PM](https://user-images.githubusercontent.com/835911/125874154-45370866-b7bf-4266-9735-78f571b236fa.png)

### Initial Page Load Errors
```
editor-modules.min.js?ver=3.3.0:2 Uncaught TypeError: Cannot set property '3' of undefined
    at editor-modules.min.js?ver=3.3.0:2
    at Array.forEach (<anonymous>)
    at Container.handleChildrenRecursive (editor-modules.min.js?ver=3.3.0:2)
    at Container.initialize (editor-modules.min.js?ver=3.3.0:2)
    at new Container (editor-modules.min.js?ver=3.3.0:2)
    at n.getContainer (editor.min.js?ver=3.3.0:2)
    at editor.min.js?ver=3.3.0:2
    at underscore.min.js?ver=1.8.3:2

20:14:17.603 common.min.js?ver=3.3.0:2 Uncaught ReferenceError: originalEvent is not defined
    at onError (common.min.js?ver=3.3.0:2)
    at dispatch (jquery.min.js?ver=3.5.1:2)
    at v.handle (jquery.min.js?ver=3.5.1:2)
```

### Click Errors
```
editor.min.js?ver=3.3.0:2 Uncaught TypeError: Cannot read property 'isEditable' of undefined
    at n.onEditRequest (editor.min.js?ver=3.3.0:2)
    at p (backbone.min.js?ver=1.4.0:2)
    at f (backbone.min.js?ver=1.4.0:2)
    at c (backbone.min.js?ver=1.4.0:2)
    at n.trigger (backbone.min.js?ver=1.4.0:2)
    at n.onEditButtonClick (editor.min.js?ver=3.3.0:2)
    at HTMLDivElement.dispatch (jquery.min.js?ver=3.5.1:2)
    at HTMLDivElement.v.handle (jquery.min.js?ver=3.5.1:2)
20:28:14.600 common.min.js?ver=3.3.0:2 Uncaught ReferenceError: originalEvent is not defined
    at onError (common.min.js?ver=3.3.0:2)
    at dispatch (jquery.min.js?ver=3.5.1:2)
    at v.handle (jquery.min.js?ver=3.5.1:2)
```

## Steps to reproduce

1. Create a post with a text box
2. Try and edit that post by clicking the pencil edit icon

## Isolating the problem
- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above. (only on certain posts, but those posts will always fail)

## Environment

<details>
<summary>System Info</summary>

```			
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Source distribution v5.7.28
	PHP Version: 7.4.15
	PHP Max Input Vars: 1000
	PHP Max Post Size: 512M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.7.2
	Site URL: https://www.abcrafty.com
	Home URL: https://www.abcrafty.com
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 300M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: America/New_York
	Debug Mode: Inactive

== Theme ==
	Name: Astra
	Version: 3.6.4
	Author: Brainstorm Force
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36

== Active Plugins ==
	Calculated Fields Form
		Version: 1.1.3
		Author: CodePeople

	Classic Editor
		Version: 1.6
		Author: WordPress Contributors

	DreamHost Panel Login
		Version: 1.0.0
		Author: DreamHost, danielbachhuber

	Elementor
		Version: 3.3.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.3.2
		Author: Elementor.com

	Enable Media Replace
		Version: 3.5.0
		Author: ShortPixel

	Featured Images in RSS for Mailchimp & More
		Version: 1.5.8
		Author: 5 Star Plugins

	HubSpot All-In-One Marketing - Forms, Popups, Live Chat
		Version: 8.3.68
		Author: HubSpot

	HubSpot for WooCommerce
		Version: 1.2.1
		Author: MakeWebBetter

	Insert Headers and Footers
		Version: 1.6.0
		Author: WPBeginner

	Jetpack
		Version: 9.9
		Author: Automattic

	jQuery Pin It Button for Images
		Version: 3.0.6
		Author: Marcin Skrzypiec

	Redirection
		Version: 5.1.1
		Author: John Godley

	ShortPixel Image Optimizer
		Version: 4.22.2
		Author: ShortPixel

	Smash Balloon Instagram Feed
		Version: 2.9.1
		Author: Smash Balloon

	UpdraftPlus - Backup/Restore
		Version: 1.16.56
		Author: UpdraftPlus.Com, DavidAnderson

	WooCommerce
		Version: 5.5.1
		Author: Automattic

	WooCommerce Shipping & Tax
		Version: 1.25.17
		Author: WooCommerce

	WooCommerce Square
		Version: 2.5.1
		Author: WooCommerce

	WP Recipe Maker
		Version: 7.4.0
		Author: Bootstrapped Ventures

	WP Rocket
		Version: 3.9.0.5
		Author: WP Media

	Yoast SEO
		Version: 16.7
		Author: Team Yoast


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	


== Elements Usage ==
	
	archive : 4
		breadcrumbs : 1
		posts : 1
		sidebar : 1
		text-editor : 1
		theme-archive-title : 1
	footer : 1
		form : 1
		heading : 1
		image : 1
		nav-menu : 2
		social-icons : 1
		text-editor : 1
	header : 1
		nav-menu : 1
		search-form : 1
		social-icons : 1
		theme-site-logo : 1
		woocommerce-menu-cart : 1
	page : 6
		breadcrumbs : 1
		button : 4
		form : 1
		heading : 9
		image : 2
		shortcode : 2
		social-icons : 3
		spacer : 3
		text-editor : 5
		woocommerce-menu-cart : 1
		woocommerce-products : 2
	popup : 4
		button : 4
		image : 4
	product : 1
		spacer : 1
		woocommerce-breadcrumb : 1
		woocommerce-product-add-to-cart : 1
		woocommerce-product-data-tabs : 1
		woocommerce-product-images : 1
		woocommerce-product-meta : 1
		woocommerce-product-price : 1
		woocommerce-product-rating : 1
		woocommerce-product-short-description : 1
		woocommerce-product-title : 1
		woocommerce-product-upsell : 1
	section : 15
		button : 8
		form : 5
		global : 3
		heading : 60
		image : 43
		menu-anchor : 1
		search-form : 1
		share-buttons : 2
		shortcode : 1
		social-icons : 1
		spacer : 11
		table-of-contents : 2
		template : 2
		text-editor : 49
		theme-archive-title : 1
		theme-post-excerpt : 1
		theme-post-featured-image : 2
		video : 1
		wc-archive-products : 1
		woocommerce-menu-cart : 1
		wprm-recipe : 2
	widget : 0
		button : 1
		share-buttons : 1
		slides : 1
		table-of-contents : 1
	wp-post : 40
		button : 57
		form : 30
		global : 20
		heading : 359
		html : 5
		image : 586
		menu-anchor : 23
		share-buttons : 32
		shortcode : 2
		spacer : 126
		table-of-contents : 12
		template : 16
		text-editor : 782
		theme-post-featured-image : 27
		video : 29
		woocommerce-products : 1
		wprm-recipe : 35
	wp-page : 8
		breadcrumbs : 2
		button : 1
		form : 2
		heading : 32
		image : 39
		image-gallery : 1
		posts : 5
		shortcode : 2
		sidebar : 4
		spacer : 12
		text-editor : 51
		video : 1
	single-post : 1
		breadcrumbs : 1
		heading : 1
		icon : 1
		post-comments : 1
		post-info : 1
		post-navigation : 1
		posts : 1
		share-buttons : 1
		sidebar : 1
		theme-post-content : 1
		theme-post-title : 1
	single-page : 2
		form : 2
		heading : 4
		image : 2
		posts : 2
		spacer : 4
		text-editor : 2
		video : 2
	product-archive : 3
		breadcrumbs : 2
		button : 4
		heading : 4
		spacer : 4
		text-editor : 2
		theme-archive-title : 1
		woocommerce-archive-description : 1
		woocommerce-products : 3
	error-404 : 1
		heading : 3
		image : 1
		posts : 1
		search-form : 1
		spacer : 1


== Log ==
	
Log: showing 20 of 382021-04-22 15:58:49 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.1.4',
  'to' => '3.2.1',
)]
2021-04-26 15:57:24 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.1',
  'to' => '3.2.2',
)]
2021-05-05 04:05:57 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.2',
  'to' => '3.2.3',
)]
2021-05-27 15:50:15 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.3',
  'to' => '3.2.4',
)]
2021-05-27 15:50:17 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.3',
  'to' => '3.2.4',
)]
2021-06-16 15:49:38 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.4',
  'to' => '3.2.5',
)]
2021-07-13 15:59:09 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.5',
  'to' => '3.3.0',
)]
2021-07-13 15:59:17 [info] elementor::elementor_updater Started 
2021-07-13 15:59:17 [info] Elementor/Upgrades - _on_each_version Start  
2021-07-13 15:59:19 [info] Elementor/Upgrades - _on_each_version Finished 
2021-07-13 15:59:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.5',
  'to' => '3.3.0',
)]
2021-07-14 19:38:07 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.1.1',
  'to' => '3.3.1',
)]
2021-07-14 19:38:09 [info] elementor-pro::elementor_pro_updater Started 
2021-07-14 19:38:09 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start  
2021-07-14 19:38:10 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Finished 
2021-07-14 19:38:10 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Start  
2021-07-14 19:38:13 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Finished 
2021-07-14 19:38:13 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.1.1',
  'to' => '3.3.1',
)]
2021-07-14 21:06:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.3.1',
  'to' => '3.3.2',
)]
2021-07-14 21:06:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.3.1',
  'to' => '3.3.2',
)]

PHP: showing 5 of 5PHP: 2021-03-14 18:33:00 [warning X 1][/home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor-pro/modules/assets-manager/asset-types/icons/custom-icons.php::180] unlink(box-128.png): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-04-16 09:09:21 [error X 1][/home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor-pro/modules/global-widget/widgets/global-widget.php::180] Uncaught Error: Call to a member function get_component() on null in /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor-pro/modules/global-widget/widgets/global-widget.php:180
Stack trace:
#0 /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor-pro/modules/global-widget/widgets/global-widget.php(68): ElementorPro\Modules\GlobalWidget\Widgets\Global_Widget->set_preview_settings(Array)
#1 /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor/includes/base/element-base.php(736): ElementorPro\Modules\GlobalWidget\Widgets\Global_Widget->get_raw_data(false)
#2 /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor/includes/base/element-base.php(736): Elementor\Element_Base->get_raw_data(false)
#3 /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor/core/base/document.php(816): Elementor\Element_Base->get_raw_data(false)
#4 /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor/modules/usage/module.php(566): Elementor\Core\Base\Document->get_elements_raw_data(Array)
#5 /home/dh_zitsi5/abcr [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-04-18 20:06:40 [notice X 6][/home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-05-09 19:06:11 [error X 4][/home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::281] Uncaught Error: Call to a member function update_meta() on bool in /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:281
Stack trace:
#0 /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(162): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /home/dh_zitsi5/abcrafty.com/wp-includes/class-wp-hook.php(292): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /home/dh_zitsi5/abcrafty.com/wp-includes/class-wp-hook.php(316): WP_Hook->apply_filters('', Array)
#5 /home/d [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-05-21 07:15:20 [error X 5][/home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor-pro/modules/global-widget/widgets/global-widget.php::180] Uncaught Error: Call to a member function get_component() on null in /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor-pro/modules/global-widget/widgets/global-widget.php:180
Stack trace:
#0 /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor-pro/modules/global-widget/widgets/global-widget.php(68): ElementorPro\Modules\GlobalWidget\Widgets\Global_Widget->set_preview_settings(Array)
#1 /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor/includes/base/element-base.php(736): ElementorPro\Modules\GlobalWidget\Widgets\Global_Widget->get_raw_data(false)
#2 /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor/includes/base/element-base.php(736): Elementor\Element_Base->get_raw_data(false)
#3 /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor/core/base/document.php(817): Elementor\Element_Base->get_raw_data(false)
#4 /home/dh_zitsi5/abcrafty.com/wp-content/plugins/elementor/modules/usage/module.php(567): Elementor\Core\Base\Document->get_elements_raw_data(Array)
#5 /home/dh_zitsi5/abcr [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]


				
```
</details>

**Comment by saumyamoon** [comment]
We are having the same issue as of 3.3.0 update

**Comment by Bovelett** [comment]
I've had this issue on prior versions, which was happening on optimized DOM output turned on. Then, on the combination of Elementor 3.2.5. (and Pro version 3.3.1) it worked fine. We were so pleased that text editing in the widget on the page was working again!

I have an active staging environment at hand to prove that it is working in 3.2.5. 

If you need the system info, please let me know?

Thanks!

- Anne

**Comment by saumyamoon** [comment]
Just to be clear, _all_ text widget editing is disabled for us in 3.3.0 - on page and in the editing sidebar.

**Comment by Bovelett** [comment]
Hey @mikedecosta Are you also using Steroids for Elementor by OoohBoi by any chance? And if so, have you copied and pasted sections from one page to another?

**Comment by mikedecosta** [comment]
> Hey @mikedecosta Are you also using Steroids for Elementor by OoohBoi by any chance? And if so, have you copied and pasted sections from one page to another?

No @Bovelett, I am not using that plugin.

**Comment by Mal027** [comment]
This was occurring when I swapped over to 3.3.0. Noticeable issues were:

- Pencil (Edit) functionality would not work on some items on some pages
- Duplicating an item would duplicate the entire section
- Some item previews would not load in the preview unless I duplicated the entire section

I reverted back to 3.2.5 as I am currently in the process of rolling out some changes to my site, so I did not want to have to deal with those bugs, but it sounds similar in nature to what you are experiencing.

**Comment by JidWictut** [comment]
Hello,

I have the same problem since the update to 3.3.0. 
Some widgets can no longer be edited. On some pages everything works and on some it doesn't.

**Comment by steve1401** [comment]
We are also experiencing this same issue. We've also found that global widgets can't be edited using elementor direct from the list of widgets and for one of our sites, the edit issue is restricted to those pages with global widgets active..

**Comment by nickdgreen** [comment]
There seems to be an issue with Global widgets. On a site with no additional plugins with a Global Button Widget in a column, the other text elements in the same column cannot be edited. If the global widget is deleted and the page saved and reloaded the elements become editable again. Additionally, if the Global Widget is dragged to move it to another column it disappears. If the widget is re-added when the page is saved, closed and then reopened there will be two instances of the global widget.

**Comment by plethoradesign** [comment]
@Bovelett, I am using Steroids for Elementor, so I went ahead and disabled that plugin and then tested by creating a new page and editing that. It made no difference. 

@nickdgreen, I can confirm it's to do with Global widgets; pages with such a widget have a slew of JS errors behind the scenes, and usually that means I can't edit my text widgets (sometimes it works though). I have also noticed the disappearing global widget issue (when moving one to another column).

**Comment by lennoxcreativeco** [comment]
Also having issues on client sites with global widgets. Lots of glitchiness, some aren't editable at all (clicking on them does nothing, right clicking just opens up the section's options menu). Issues with sliders not respecting column sizes as well.

**Comment by Mal027** [comment]
> Also having issues on client sites with global widgets. Lots of glitchiness, some aren't editable at all (clicking on them does nothing, right clicking just opens up the section's options menu). Issues with sliders not respecting column sizes as well.

I believe it was mentioned that they plan on releasing a series of **hotfixes** to address the global widget issue.

As for the column sizes, in another article, it was mentioned this was due to a bug with the latest version of Jetpack. Jetpack is aware and is releasing a fix this week!

**Comment by lennoxcreativeco** [comment]
> > Also having issues on client sites with global widgets. Lots of glitchiness, some aren't editable at all (clicking on them does nothing, right clicking just opens up the section's options menu). Issues with sliders not respecting column sizes as well.
> 
> I believe it was mentioned that they plan on releasing a series of **hotfixes** to address the global widget issue.
> 
> As for the column sizes, in another article, it was mentioned this was due to a bug with the latest version of Jetpack. Jetpack is aware and is releasing a fix this week!

Awesome, thank you for the update!

**Comment by steve1401** [comment]
From a separate email support conversation with Elementor, they have replied to say that this is a bug with a hotfix coming soon. Their response to me:

> Our Tier 3 support team has confirmed that this is a bug that will be addressed in upcoming updates. This only happens in Elementor 3.3.0 with version 3.3.2 of Pro from what we're seeing when trying to edit a non-global widget when a global widget is present in the same column. Downgrading back to Elementor 3.2.5 should help especially if you need to make changes at this time.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.3.1

Feel free to update, please update if this issue was resolved on your end

Thanks!

**Comment by lennoxcreativeco** [comment]
> This issue has been resolved in Elementor v3.3.1
> 
> Feel free to update, please update if this issue was resolved on your end
> 
> Thanks!

I'm not OP but was also having issues on client sites. I updated one and I can change the background on an inner section that contained global widgets now (it wasn't allowing me to select the inner section at all before), however still having issues specifically with the testimonial widget that is set as a global widget. When I open the widget it's not showing any of my testimonials (just the default filler that's in there when you set up the widget) but my testimonials show on the actual site. Also can't edit one of the global testimonial widgets I have at all (screenshot to show what happens when I click it).

![Screen Shot 2021-07-20 at 8 57 12 AM](https://user-images.githubusercontent.com/87677223/126356861-fa3e5394-7aae-4240-937f-1d09d18a29f7.png)

**Comment by caspervoogt** [comment]
I am running WP 5.8, Elementor 3.3.1 and Elementor Pro 3.3.3 and I still can't move Global Widgets from column to column. I can *add* one to a column; I just can't move them. [Here's a screencast showing the issue](https://www.loom.com/share/34f6a94ceb4e49aa88512ab7535762ff)

**Comment by innugi** [comment]
same here!

**Comment by mikedecosta** [comment]
> This issue has been resolved in Elementor v3.3.1
> 
> Feel free to update, please update if this issue was resolved on your end
> 
> Thanks!

Still an issue for me even after updating to `Elementor 3.3.1` and `Elementor Pro 3.3.4`

**Comment by shilo-ey** [comment]
@innugi @mikedecosta 

Please contact our support at support @elementor.com for guidance in resolving this issue and mention that you were directed by me (shilo-ey) and reference this issue.

thanks

**Comment by webinfinitynz** [comment]
I'm also having the same issue, but on various widgets (Posts, Buttons etc). When I try to select the widget in the editor, it doesn't select and I get a load of 'Uncaught TypeError: this.container is undefined' errors in the dev tools console.
I'm editing an Archive in the Template Builder and it has global widgets.

It was all fine (and I haven't added any new plugins) up until recently. I'm not 100% when but I think since a recent Elementor release.

The site has Elementor 3.3.1 and Elementor Pro 3.3.4

**Comment by greggh** [comment]
We are having the same issue. We are not using any other plugins that add-on to Elementor (like the ones mentioned in this thread already).

**Comment by shilo-ey** [comment]
> @innugi @mikedecosta
> 
> Please contact our support at support @elementor.com for guidance in resolving this issue and mention that you were directed by me (shilo-ey) and reference this issue.
> 
> thanks

We need to identify the issue. please share with us credentials for better clarity.

**Comment by Pieroket** [comment]
Hi,
We are facing the same issue with Elementor `3.4.0` and Elementor Pro `3.3.7`.
For some reason, it does not affect all the pages.

```js
editor-modules.min.js?ver=3.4.0:2 Uncaught TypeError: Cannot set property children of #<Container> which has only a getter
    at Container.handleChildrenRecursive (editor-modules.min.js?ver=3.4.0:2)
    at editor-modules.min.js?ver=3.4.0:2
    at Array.forEach (<anonymous>)
    at Container.handleChildrenRecursive (editor-modules.min.js?ver=3.4.0:2)
    at Container.initialize (editor-modules.min.js?ver=3.4.0:2)
    at new Container (editor-modules.min.js?ver=3.4.0:2)
    at n.getContainer (editor.min.js?ver=3.4.0:2)
    at editor.min.js?ver=3.4.0:2
    at underscore.min.js?ver=1.8.3:2

common.min.js?ver=3.4.0:2 Uncaught ReferenceError: originalEvent is not defined
    at onError (common.min.js?ver=3.4.0:2)
    at dispatch (jquery.min.js?ver=3.6.0:2)
    at v.handle (jquery.min.js?ver=3.6.0:2)
```

Should we contact the support?

**Comment by Chertii** [comment]
I can't edit anything within inner sections that contain a global widget.

The bug occurs on all websites - no plugins except Elementor & Elementor Pro loaded.

Deleting the global widget would solve the problem. Please fix this bug!

WP 5.8.1
Elementor 3.4.3
Elementor Pro 3.4.1

**Comment by geoffnori** [comment]
Wanted to also contribute to this thread. We are having the same issue. Seems to be 100% related to global widgets. If you have ANY global widgets inside of a section, the widgets within the section become un-editable because of the error others are reporting.

Deleting the global widgets or unlinking them fixes it.

**Comment by Mal027** [comment]
This issue has worsened as of today. Now, even though my Global Widgets are their own inner sections (which resolved the issue before), on some pages, sections are uneditable. Now, I used global scroll ups, user menus per page so unlinking is not an option. In addition, I _did_ try to unlink one of my scroll-up buttons, but it reverted that, and my global widget version to its default format, removing all changes I made to it. 

Duplicating the section every time I want to edit something is a temporary solution, but this is an ongoing issue that is becoming rather irritating.
